x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/content.test.tsx

```diff
@@ -40,4 +41,16 @@ describe('AttacksPageContent', () => {
       expect(screen.getByTestId('schedule')).toBeInTheDocument();
     });
   });
+
+  it('should render `Assignee` button', async () => {
```

Comment:
Leaving a comment but feel free to ignore it: I feel like this test and the previous one should be handled within the very first one (`should render correctly`). Checking that things on the page are rendered is exactly what the first tests is doing...